physionet-django/user/enums.py

```diff
@@ -19,3 +19,15 @@ class RequiredField(IntEnum):
     @classmethod
     def choices(cls):
         return tuple((option.value, option.name) for option in cls)
+
+
+class EventType(IntEnum):
+    COURSE = 0
+    DATATHON = 1
+    CONFERENCE = 2
```

Comment:
I don't think there is a need for all of these types now. Perhaps just limit to course and workshop?